assets/xml/objects/object_test3.xml

```diff
@@ -1,78 +1,88 @@
 ﻿<Root>
+    <!-- Object for Kafei -->
     <File Name="object_test3" Segment="6">
+        <!-- Textures -->
         <Texture Name="object_test3_TLUT_000000" OutName="tlut_000000" Format="rgba16" Width="16" Height="16" Offset="0x0" />
         <Texture Name="object_test3_TLUT_000200" OutName="tlut_000200" Format="rgba16" Width="16" Height="16" Offset="0x200" />
-        <Texture Name="object_test3_Tex_000400" OutName="tex_000400" Format="rgba16" Width="16" Height="16" Offset="0x400" />
-        <Texture Name="object_test3_Tex_000600" OutName="tex_000600" Format="rgba16" Width="16" Height="16" Offset="0x600" />
-        <Texture Name="object_test3_Tex_000800" OutName="tex_000800" Format="ci8" Width="8" Height="8" Offset="0x800" />
-        <Texture Name="object_test3_Tex_000840" OutName="tex_000840" Format="ci8" Width="8" Height="16" Offset="0x840" />
-        <Texture Name="object_test3_Tex_0008C0" OutName="tex_0008C0" Format="ci8" Width="16" Height="16" Offset="0x8C0" />
-        <Texture Name="object_test3_Tex_0009C0" OutName="tex_0009C0" Format="ci8" Width="32" Height="32" Offset="0x9C0" />
-        <Texture Name="object_test3_Tex_000DC0" OutName="tex_000DC0" Format="ci8" Width="64" Height="32" Offset="0xDC0" />
-        <Texture Name="object_test3_Tex_0015C0" OutName="tex_0015C0" Format="ci8" Width="8" Height="16" Offset="0x15C0" />
-        <Texture Name="object_test3_Tex_001640" OutName="tex_001640" Format="ci8" Width="16" Height="16" Offset="0x1640" />
+        <Texture Name="object_test3_TLUT_000400" OutName="tlut_000400" Format="rgba16" Width="16" Height="16" Offset="0x400" />
+        <Texture Name="gKafeiEyesTLUT" OutName="kafei_eyes_tlut" Format="rgba16" Width="16" Height="16" Offset="0x600" />
+        <Texture Name="gKafeiSkinTex" OutName="kafei_skin" Format="ci8" Width="8" Height="8" Offset="0x800" />
+        <Texture Name="gKafeiHair1Tex" OutName="kafei_hair1" Format="ci8" Width="8" Height="16" Offset="0x840" />
+        <Texture Name="gKafeiNoseTex" OutName="tex_0008C0" Format="ci8" Width="16" Height="16" Offset="0x8C0" />
+        <Texture Name="gKafeiMouthClosedTex" OutName="kafei_mouth_closed" Format="ci8" Width="32" Height="32" Offset="0x9C0" />
+        <Texture Name="gKafeiEyesOpenTex" OutName="kafei_eyes_open" Format="ci8" Width="64" Height="32" Offset="0xDC0" />
+        <Texture Name="gKafeiHair2Tex" OutName="kafei_hair2" Format="ci8" Width="8" Height="16" Offset="0x15C0" />
+        <Texture Name="gKafeiFingersTex" OutName="kafei_eyes_fingers" Format="ci8" Width="16" Height="16" Offset="0x1640" />
         <Texture Name="object_test3_Tex_001740" OutName="tex_001740" Format="ci8" Width="32" Height="32" Offset="0x1740" />
```

Comment:
This is the texture for his boot. You can tell because it's only used in `gKafeiLeftFootDL` and `gKafeiRightFootDL`.